### ISSUE TITLE
Fails with Go and other C like languages

### ISSUE THREAD

**Issue by bloudraak** [tomdalling/jekyll-include_snippet#1]
It seems that the plugin fails for any C like language.

Here's an example document:

````
---
title: Hello
layout: page
---

# Hello

```go
{% include_snippet my_method_snippet from code/hello.go %}
```
````
The code for `code/hello.go` looks something like this

	// #begin-snippet: my_method_snippet

	package main

	import "fmt"

	func main() {
		fmt.Println("Hello World")
	}

	// #end-snippet

When it is rendered, it looks like this:

```
ackage main
mport "fmt"
unc main() {
fmt.Println("Hello World")
```

Instead, it should look something like this:

	package main

	import "fmt"

	func main() {
		fmt.Println("Hello World")
	}

**Comment by tomdalling** [comment]
It's chopping off the first 3 characters because it treats everything before the `#` character as indentation. I can probably fix this by allowing `//` for comments in addition to `#`. I'll have a look when I get a chance.

**Comment by tomdalling** [comment]
This is implemented in v0.2.0. Let me know how it goes for you.
